lib/compilers/_all.ts

```diff
@@ -114,3 +114,4 @@ export {ZigCC} from './zigcc.js';
 export {ZigCompiler} from './zig.js';
 export {ZigCXX} from './zigcxx.js';
 export {z88dkCompiler} from './z88dk.js';
+export {JavascriptCompiler} from './v8.js';
```

Comment:
Can you keep this in alphabetical order please

Reply:
Yep! Making this change.